index.html

```diff
@@ -18,7 +18,7 @@ <h1 class="tagline col-md-5">
             <i class="fa fa-plus"></i>
             <img src="{{ site.baseurl }}/assets/images/foreman.png" alt="Foreman"/>
           </p>
-          <a href="{{ site.baseurl }}/docs/2.0/installation/index.html" class="get-started">
+          <a href="{{ site.baseurl }}/docs/{{ site.latest }}/installation/index.html" class="get-started">
```

Comment:
@jlsherrill `latest` is set in _config.yml, so you would only update that on GA announcement (like here).  You can add KATELLO-2.2 branch w/out affecting this.